module-3/pop/HomePage.js

```diff
@@ -2,4 +2,30 @@
  * Create HomePage class representing the EPAM.com home page.
  * Add main widgets and element of the page and write tests
  * for it (test/pop/HomePage.spec.js).
- */
\ No newline at end of file
+ */
+const Layout = require("./Layout")
+const ElementFinder = require("../test/mock/ElementFinder");
```

Comment:
It is enough to import the `element `static method, and it will create the instance of the `ElementFinder`